packages/sources/coinmetrics/src/transport/realized-vol.ts

```diff
@@ -0,0 +1,86 @@
+import { BaseEndpointTypes, ResponseData } from '../endpoint/realized-vol'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+
+const logger = makeLogger('Coinmetrics RealizedVol HTTP')
+
+interface RealizedVolResponseBody {
+  data: (Record<string, number> & {
+    asset: string
+    time: string // DateTime (e.g. '2022-12-08T23:50:00.000000000Z')
```

Comment:
Can't this be `time: Date`?

Reply:
Sadly no, because that will just make TypeScript think it's a date object when it is still just a `string` and so calling any Date functions on it will fail at runtime.